sdk/extensions/Azure.AspNetCore.DataProtection.Blobs/tests/DataProtectionBlobFunctionalTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace Azure.AspNetCore.DataProtection.Blobs.Tests
+{
+    public class DataProtectionBlobFunctionalTest
+    {
+        private static readonly string StorageAccountName = Environment.GetEnvironmentVariable("BLOB_STORAGE_ACCOUNT_NAME");
+        private static readonly string StorageAccountKey = Environment.GetEnvironmentVariable("BLOB_STORAGE_ACCOUNT_KEY");
+
+        [Test]
+        [Category("Live")]
```

Comment:
`[LiveOnly]`?

Reply:
Can't reference the test framework from this test project for a complicated reason that basically comes down to source references suck.